### ISSUE TITLE
[BUG] Incompatible with Foundry V12

### ISSUE THREAD

**Issue by kiloforce** [mclemente/compendium-browser#18]
![image](https://github.com/user-attachments/assets/194fe1d5-ee89-48a6-b919-a9f82dffc325)
Error: [undefined] module compendium-browser version 2.2.7 requires an older version of the Foundry Virtual Tabletop software: 11 or older

Do you have plans to support Foundry V12?

**Comment by mclemente** [comment]
I was working on a rewrite of the module (see #12), but then D&D5e 3.3 added its own Compendium Browser, so I don't see any reason to update it.

EDIT: If anyone reading this wants to make a fork, I suggest you pick up the Compendium Browser from dnd5e's project instead, since it won't have pre-V12 code and will be written in AppV2.

**Comment by kiloforce** [comment]
Makes sense.  Thank you for your past efforts.  It has been super helpful.
